src/components/organize/campaigns/CampaignActionButtons.tsx

```diff
@@ -64,46 +70,39 @@ const CampaignActionButtons: React.FunctionComponent<CampaignActionButtonsProps>
                 </Link>
             </Box>
             <Box>
-                <Button
-                    color="secondary"
-                    data-testid="campaign-action-buttons-menu-activator"
-                    disableElevation
-                    onClick={ (e) => setMenuActivator(e.currentTarget) }
-                    variant="contained">
-                    <Settings />
-                </Button>
+                <ZetkinEllipsisMenu items={ [
```

Comment:
Is it better to put it here than to use the `ellipsisMenuItems` property of `TabbedLayout` (inside `SingleCampaignLayout`)?